lte/gateway/c/session_manager/SessionState.cpp

```diff
@@ -1046,24 +1050,26 @@ bool SessionState::receive_monitor(
       update.credit().level() == MonitoringLevel::SESSION_LEVEL) {
     update_session_level_key(update, update_criteria);
   }
-  auto it = monitor_map_.find(update.credit().monitoring_key());
+  auto mkey = update.credit().monitoring_key();
+  auto it = monitor_map_.find(mkey);
   if (it == monitor_map_.end()) {
     // new credit
     return init_new_monitor(update, update_criteria);
   }
-  auto credit_uc =
-      get_monitor_uc(update.credit().monitoring_key(), update_criteria);
+  auto credit_uc = get_monitor_uc(mkey, update_criteria);
   if (!update.success()) {
     it->second->credit.mark_failure(update.result_code(), credit_uc);
     return false;
   }
-  const auto& gsu = update.credit().granted_units();
-  MLOG(MINFO) << session_id_ << " Received monitor credit for "
-              << update.credit().monitoring_key();
-  FinalActionInfo final_action_info;
-  it->second->credit.receive_credit(gsu, credit_uc);
+
   if (update.credit().action() == UsageMonitoringCredit::DISABLE) {
-    monitor_map_.erase(update.credit().monitoring_key());
+    MLOG(MINFO) << "Erasing monitor " << mkey << " from DISABLE instruction";
```

Comment:
Yeah, I think this is OK to leave at info because it's a relatively important state change that only happens once for each monitor. If it gets too noise/not useful we can bump it down. :)

Reply:
The problem I see is INFO level is for errors only and general information from the service. Messages coming from a specific user that are not errors shouldn't be at the info. But I understand that this information may be valuable now that we want to make sure the bug was fixed and the logs we receive are from INFO level. 

Maybe we can move it into DEBUG once we are sure its fixed?